### DISCUSSION TITLE
Some emojis don't render correctly

### DISCUSSION THREAD

**Discussion by fulalas** [d99kris/nchat#225] [Q&A]
Using this emoji in Telegram `:thumbsup::light_skin_tone:` I have this output (note the odd square):

![bug](https://github.com/d99kris/nchat/assets/27843666/6996e613-fd55-4d8a-a591-dbf54bd29225)

Tried in xfce4-terminal `1.1.3` and alacritty `0.10.0`.

nchat `4.75`, but this bug exists since long time ago.

**Comment by d99kris** [comment]
If you copy this emoji 👍🏻 from your web browser and paste into the terminal shell or text editor (ex: nano, emacs, vim), does it render correctly?

**Comment by fulalas** [comment]
Hmmm... It doesn't render correctly in any of these scenarios. Do you know a way to fix that?

**Comment by d99kris** [comment]
I don't know. I mainly use `Terminal` on macOS and `gnome-terminal` on Ubuntu, using font Monaco (but not sure it matters) and in those environments this particular emoji renders fine (in nchat, shell and terminal-based editors). 

I do have encountered _other_ emojis not rendering correctly, but I don't recall exactly which ones.

Since nchat and other terminal-based apps behave similarly I don't think this is an nchat bug. I'll convert the issue to a Q&A discussion item instead and leave it open for a while, in case someone else have some idea on how to solve it.
